README.md

```diff
@@ -1,15 +1,20 @@
 ![PODMAN logo](https://raw.githubusercontent.com/containers/common/main/logos/podman-logo-full-vert.png)
 
 # Podman: A tool for managing OCI containers and pods
+![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)
+![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/containers/podman)
+[![Go Report Card](https://goreportcard.com/badge/github.com/containers/libpod)](https://goreportcard.com/report/github.com/containers/libpod)
+
+<br/>
 
 Podman (the POD MANager) is a tool for managing containers and images, volumes mounted into those containers, and pods made from groups of containers.
 Podman runs containers on Linux, but can also be used on Mac and Windows systems using a Podman-managed virtual machine.
 Podman is based on libpod, a library for container lifecycle management that is also contained in this repository. The libpod library provides APIs for managing containers, pods, container images, and volumes.
 
-* [Latest Version: 4.4.0](https://github.com/containers/podman/releases/tag/v4.4.0)
+<!-- * [Latest Version: 4.4.4](https://github.com/containers/podman/releases/tag/v4.4.4)
```

Comment:
@R3DRUN3 do we need to keep this commented out? Guess we could just remove it? Could you please also squash commits?

Reply:
@lsm5 done.

Reply:
@lsm5 yeah sure, done! 👍